rust/hyperlane-core/src/traits/pending_operation.rs

```diff
@@ -114,7 +119,7 @@ pub trait PendingOperation: Send + Sync + Debug + TryBatchAs<HyperlaneMessage> {
     fn set_retries(&mut self, retries: u32);
 }
 
-#[derive(Debug, Display, Clone)]
+#[derive(Debug, Display, Clone, Serialize, Deserialize, PartialEq)]
 /// Status of a pending operation
 pub enum PendingOperationStatus {
```

Comment:
Can we note here and ReprepareReason / ConfirmReason that we should only add new variants otherwise we break reading from the DB?

Reply:
💯